### DISCUSSION TITLE
tooltip for mantine component (checkbox etc.)

### DISCUSSION THREAD

**Discussion by NIkitaTemerko** [KevinVandy/mantine-react-table#64] [Q&A] [answered]
mrt force using tooltip for checkboxes, radio buttons, expand button. I want to remove it, but I cant find prop which can do it, or at least translate it into another lng or allows to change the content.

**Comment by KevinVandy** [comment]
Hoping you've at least read the Localization Docs: https://www.mantine-react-table.com/docs/guides/localization

**Reply by NIkitaTemerko** [reply]
@KevinVandy  I missed it, thx! But still, is it possible to remove the tooltip?

**Reply by KevinVandy** [reply, marked as answer]
I think I have it set so that you can set title props to undefined and it will disable the tooltips. The tooltips are there for good accessibility so just ensure you have that taken care of before disabling them.

**Reply by NIkitaTemerko** [reply]
@KevinVandy Thx! Though at this moment it looks like title undefined reset it to default. But anyway I can at least set there more precise info or change it to smth else. I checked it into StackBlitz
```
mantineSelectCheckboxProps={{
  title: 'pop',
}}
mantineSelectAllCheckboxProps={{
  title: undefined,
}}
```
<img width="242" alt="Screenshot 2023-06-01 at 20 22 25" src="https://github.com/KevinVandy/mantine-react-table/assets/99005728/10ef5d09-2ccd-43cf-afaa-47bfc4df98e6">
<img width="96" alt="Screenshot 2023-06-01 at 20 22 36" src="https://github.com/KevinVandy/mantine-react-table/assets/99005728/10b9ab9c-8d01-456a-baf3-11d64543555c">

Thx once more

**Comment by Wiwida** [comment]
Same problem here, undefined not work.
